app/assets/stylesheets/layout_components/buttons/index.scss

```diff
@@ -1,4 +1,4 @@
-@import "button-grouping";
 @import "buttons";
+@import "button-grouping";
```

Comment:
did you check this in any other apps to make sure it doesnt affect anything?

Reply:
yes, it fixes the hover state in all other apps. They weren't working anywhere before.